source/custom_templates/_tx_api_page.txt

```diff
@@ -1,8 +1,9 @@
 .. include:: _:page:.tmp
 
-.. include:: common_response_params.rst
+.. include:: _:org_type:_param_table.rst
 
-.. include:: optional_fields.txt
+The following fields are optional based on your account type:
 
+.. include:: _:org_type:_optional_param_table.rst
 
```

Comment:
Cool, if we can make all the optional fields still look coherent and flow nicely in the final doc output page, I think it's worth it to reduce duplication in a lot of spots.